docs/source/install.rst

```diff
@@ -414,6 +414,14 @@ All cached entries can be viewed using ``cmake -LAH`` from a build directory.
     Path to an existing directory used to cache :file:`proj.db` to speed-up
     subsequent builds without modifications to source SQL files.
 
+.. option:: EMBED_PROJ_DATA_PATH
+
+    .. versionadded:: 9.5
+
+    Embed ``PROJ_DATA_PATH`` data files location as an alternative paths for
```

Comment:
Sorry for asking. But I do not understand the meaning of. "data files location as an alternative paths for external libraries".

I was not aware until now of a variable called `PROJ_DATA_PATH`, but only about `PROJ_DATA`.

If I understand correctly, currently the variable `PROJ_DATA` has a hardcoded default value. You want to give an option (off by default) to not set that hardcoded value, leaving the responsibility to the final user.
`PROJ_DATA` is about configuration and resource files like `proj.db` and grid files, not about external libraries.